lib/open_food_network/packing_report.rb

```diff
@@ -61,7 +61,7 @@ def rules
                               proc { |_line_items| "" },
                               proc { |_line_items| I18n.t('admin.reports.total_items') },
                               proc { |_line_items| "" },
-                              proc { |line_items| line_items.sum(&:quantity) },
+                              proc { |line_items| line_items.to_a.sum(&:quantity) },
```

Comment:
`line_items.sum(:quantity)`?

Reply:
same as above, tests where line_items is given as an array...